core/src/test/java/model/bank/CreditTest.java

```diff
@@ -19,7 +21,8 @@ public class CreditTest {
 
 	@BeforeEach
 	public void setUp() {
-		bankAccount = new BankAccount(new BigDecimal("0.05"), BigDecimal.valueOf(100));
+		bankAccount = mock(BankAccount.class);
```

Comment:
tu wystarczy to raz usawić przed @BeforeEach jako zwykłe pole w klasie, nie ma potrzeby jego aktualizowania